taipy/core/_core.py

```diff
@@ -16,7 +16,7 @@
 
 
 class Core:
-    """Deprecated. Use the `Orchestrator^` service class instead."""
+    """NOT DOCUMENTED"""
```

Comment:
The `Core` is not in the refman anymore?

Reply:
Nop. We keep it for backward compatibility, but we don't want to expose it in the 4.0 version of the doc.